server/src/test/java/org/apache/kafka/server/EligibleLeaderReplicasIntegrationTest.java

```diff
@@ -432,16 +383,16 @@ public void testLastKnownLeaderShouldBeElectedIfEmptyElr() throws ExecutionExcep
                         return false;
                     }
                 },
-                () -> String.format("Partition metadata for %s is not correct", testTopicName),
-                DEFAULT_MAX_WAIT_MS, 100L
+                DEFAULT_MAX_WAIT_MS,
+                () -> String.format("Partition metadata for %s is not correct", testTopicName)
             );
         } finally {
-            restartDeadBrokers(false);
+            clusterInstance.restartDeadBrokers();
```

Comment:
This helper method is very simple, so perhaps we could move it into the test class

Reply:
@chia7712 
Thanks for the suggestions. I've addressed problems mentioned above.